### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://hosted.weblate.org/projects/gnucash/gnucash/) for GnuCash/GnuCash.

Current translation status:

Weblate translation status

### CODE REVIEW

**Comment by fellen** [comment]
@MocioF can you review the it commit?
I have the impression, you will need a typografical convention and we have probably to add  terms with different wriing conventions like eMail, e-mail… to the glossary.

**Comment by fellen** [comment]
For the surprised:
We are experimenting with the integration of weblate. This is the first PR by weblate and wehave to develope strategies, how to handle them. Sray patient.

Until now we kept the responsible  person as `Last Translator`, But weblate takes it literally.
If there is no hidden option, we have to store the responsible person somewhere else.

**Comment by fellen** [comment]
The current Make **target check** does not make much sense, If po files are changed, the failure of 
- `msgfmt -c --check-accelerators="_" *.po` should throw a _warning_,
- `msgfmt -c *.po` should throw an _error_,

File: po/nb.po [hunk 1]
```diff
@@ -7607,13 +7610,12 @@ msgstr ""
...
 #. See http://en.wikipedia.org/wiki/ISIN for hints.
 #: gnucash/gnome-utils/dialog-commodity.c:173
-#, fuzzy
 msgid ""
 "\n"
 "Exchange code (ISIN, CUSIP or similar): "
 msgstr ""
 "\n"
-"Børskode (CUSIP eller lignende):"
+"Børskode (ISIN, CUSIP eller lignende): "
```
Comment (fellen) [inline po/nb.po:7618]: ask Verdipapirsentralen (VPS) ASA, if a norwegian variant exists.

File: po/nb.po [hunk 2]
```diff
@@ -15336,9 +15333,8 @@ msgid "<b>Quote Source Information</b>"
 msgstr "<b>Informasjon om noteringskilde</b>"
 
 #: gnucash/gtkbuilder/dialog-commodity.glade:238
-#, fuzzy
 msgid "<b>Security Information</b>"
-msgstr "Verdipapirinformasjon"
+msgstr "Sikkerhetsinfo</b>"
```
Comment (fellen) [inline po/nb.po:15337]: - Verdipapir seems closer than Sikkerhet
- starting "<b>" missing

**Comment by MocioF** [comment]
Hi Frank,
tonight I will make a PR for a new it.po in github.I have looked at weblate
and I think it can be a good solution. I think I should write or rewrite
the glossary and, about conventions, I have tried to follow some rules (as
using always email and not e-mail, mail or E-Mail).
When I started translating gnucash, I was looking for a team or
contributors but no one offered, so I thought it would have been just a
waste of time to fix the glossary (there are many mistakes in it especially
in the business features terms, not just in typo conventions). Right now,
for me to create a glossary is not a real problem (just need to find the
time) because I have followed the language choices I made in all the
strings, so after 4.3 I will spend some time in fixing the glossary ready
to be used by other contributors coming from weblate (I hope).
More, I think that all translations suggested on weblate should be first
approved by one or more guys responsible for each language. I candidate for
Italian, of course. However, I think that no translation's suggestion
should be accepted without a review by the "translator team leader" or
similar.

See you later on irc

Il giorno mar 15 dic 2020 alle ore 02:08 Frank H. Ellenberger <
notifications@github.com> ha scritto:

File: po/nb.po [hunk 3]
```diff
@@ -21010,11 +21001,11 @@ msgstr "Vis priskolonnen"
...
 #. Translators: This is a ngettext(3) message, %d is the number of prices added
 #: gnucash/import-export/csv-imp/assistant-csv-price-import.cpp:1881
-#, fuzzy, c-format
+#, c-format
 msgid "%d added price"
 msgid_plural "%d added prices"
-msgstr[0] "Legg til en ny pris."
-msgstr[1] "Legg til en ny pris."
+msgstr[0] "% tillagt pris"
+msgstr[1] "%s tillagte priser"
```
Comment (fellen) [inline po/nb.po:21008]: `%d`

Other errors, which I got, but you did not neccesary have caused them: msgfmt -c --statistics --check-accelerators="_" nb.po
nb.po:3520: msgstr lacks the keyboard accelerator mark '_'
nb.po:3633: msgstr lacks the keyboard accelerator mark '_'
nb.po:3914: msgstr lacks the keyboard accelerator mark '_'
nb.po:3992: msgstr lacks the keyboard accelerator mark '_'
nb.po:4093: msgstr lacks the keyboard accelerator mark '_'
nb.po:4405: msgstr lacks the keyboard accelerator mark '_'
nb.po:4413: msgstr lacks the keyboard accelerator mark '_'
nb.po:4425: msgstr lacks the keyboard accelerator mark '_'
nb.po:4663: msgstr lacks the keyboard accelerator mark '_'
nb.po:5427: msgstr lacks the keyboard accelerator mark '_'
nb.po:5432: msgstr lacks the keyboard accelerator mark '_'
nb.po:6842: msgstr lacks the keyboard accelerator mark '_'
nb.po:8730: msgstr lacks the keyboard accelerator mark '_'
nb.po:8820: msgstr lacks the keyboard accelerator mark '_'
nb.po:13016: msgstr lacks the keyboard accelerator mark '_'
nb.po:14859: msgstr lacks the keyboard accelerator mark '_'
nb.po:14934: msgstr lacks the keyboard accelerator mark '_'
nb.po:15011: msgstr lacks the keyboard accelerator mark '_'
nb.po:16629: msgstr lacks the keyboard accelerator mark '_'
nb.po:16815: msgstr lacks the keyboard accelerator mark '_'
nb.po:17014: msgstr lacks the keyboard accelerator mark '_'
nb.po:17029: msgstr lacks the keyboard accelerator mark '_'
nb.po:18025: msgstr lacks the keyboard accelerator mark '_'
nb.po:18029: msgstr lacks the keyboard accelerator mark '_'
nb.po:18155: msgstr lacks the keyboard accelerator mark '_'
nb.po:19095: msgstr lacks the keyboard accelerator mark '_'
nb.po:19260: msgstr lacks the keyboard accelerator mark '_'
nb.po:19277: msgstr lacks the keyboard accelerator mark '_'
nb.po:21007: format specifications in 'msgid_plural' and 'msgstr[0]' for argument 1 are not the same
msgfmt: found 30 fatal errors
2837 translated messages, 1806 fuzzy translations, 902 untranslated messages.

File: po/nb.po (hunk 1 above)
Comment (comradekingu) [reply po/nb.po:7618]: There is https://www.isin.org/norway-isin/ for what imagine is foreign citizens.
It is marked as needing edit in Weblate.

File: po/nb.po (hunk 2 above)
Comment (comradekingu) [reply po/nb.po:15337]: Consistent even. I can add some context there if you make https://hosted.weblate.org/user/kingu/#managed an admin.

File: po/nb.po (hunk 3 above)
Comment (comradekingu) [reply po/nb.po:21008]: I added some that were missing, but there are many more.

**Comment by jralls** [comment]
Resolved the conflict in it.po and merged all translations.

File: po/nb.po (hunk 1 above)
Comment (fellen) [reply po/nb.po:7618]: The ISIN is the international SIN, CUSIP is the national SIN in US, WKN in DE … If no NO variant exists, I would remove ", CUSIP".

File: po/nb.po [hunk 4]
```diff
@@ -17420,7 +17410,7 @@ msgstr "Vinduer"
 #: gnucash/gtkbuilder/dialog-preferences.glade:3638
 #, fuzzy
```
Comment (fellen) [inline po/nb.po:17411]: Still not ready?

File: po/nb.po (hunk 2 above)
Comment (fellen) [reply po/nb.po]: > I can add some context there if you make https://hosted.weblate.org/user/kingu/#managed an admin.

Granted to help with the configuration. And you have become the revisor for nb.

**Pull request closed without merging**